### PR TITLE
Send visit updated notifications via the sendBookingNotification usecase

### PR DESCRIPTION
**Pull request by jivdhaliwal**
# What
Updates the sendBookingNotification usecase to be able to send 'visit updated' notifications as well as 'new visit' notifications.

# Why
Allows us to use this usecase in the upcoming update-a-visit API to send notifications

# Screenshots

# Notes

### CODE REVIEW

File: src/usecases/sendBookingNotification.js [hunk 1]
```diff
@@ -69,4 +78,21 @@ const sendBookingNotification = ({
...
+
+  if (notificationType == NEW_NOTIFICATION) {
+    textMessageTemplateId = TemplateStore().firstText.templateId;
+    emailTemplateId = TemplateStore().firstEmail.templateId;
+  }
+
+  if (notificationType == UPDATED_NOTIFICATION) {
+    textMessageTemplateId = TemplateStore().updatedVisitText.templateId;
+    emailTemplateId = TemplateStore().updatedVisitEmail.templateId;
+  }
```
Comment (Robware) [inline src/usecases/sendBookingNotification.js:93]: How about converting this to a switch statement, then throwing an exception on the default case for when it's passed something not recognised?

File: src/usecases/sendBookingNotification.test.js [hunk 2]
```diff
@@ -29,6 +32,27 @@ describe("sendBookingNotification", () => {
...
+    emailUpdatedTemplateId = TemplateStore().updatedVisitEmail.templateId;
+  });
+
+  afterEach(() => {
+    process.env.SMS_INITIAL_TEMPLATE_ID = "";
+    process.env.SMS_UPDATED_VISIT_TEMPLATE_ID = "";
+    process.env.SMS_JOIN_TEMPLATE_ID = "";
+    process.env.EMAIL_INITIAL_TEMPLATE_ID = "";
+    process.env.EMAIL_UPDATED_VISIT_TEMPLATE_ID = "";
+    process.env.EMAIL_JOIN_TEMPLATE_ID = "";
```
Comment (Robware) [inline src/usecases/sendBookingNotification.test.js:55]: If the `TemplateStore` is being injected now, would it not be possible to mock it out so we don't need to reset the env vars between each test?

Comment (jivdhaliwal) [reply src/usecases/sendBookingNotification.test.js:55]: I decided not to go down the route of injecting the template store via the app container in the end and went for simpler implementation (just changing it to a function and setting env variables in the test)

File: src/usecases/sendBookingNotification.js (hunk 1 above)
Comment (jivdhaliwal) [reply src/usecases/sendBookingNotification.js:93]: Yep good idea, that keeps all the notification type logic in one place

**Pull request merged**